src/components/LocalNav.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Box, ListItemIcon, ListItemText, MenuItem, MenuList, Paper, Typography } from '@mui/material';
+import { LOCAL_NAVIGATION_DEFAULT_WIDTH } from '../constants/styles';
+
+export interface LocalNavMenuItem {
```

Comment:
後々コンポーネントの型定義を切り出す予定ならコメントアウトしといた方が良い
```suggestion
// TODO 型ファイルに切り出す
export interface LocalNavMenuItem {
```